docs/reference/NuGet-Client-SDK.md

```diff
@@ -57,6 +57,12 @@ Search for "json" packages using the [NuGet V3 Search API](../api/search-query-s
 
 [!code-csharp[SearchPackages](~/../nuget-samples/NuGetProtocolSamples/Program.cs?name=SearchPackages)]
 
+### Work with authenticated feeds
+
+Use [`NuGet.Protocol`](https://www.nuget.org/packages/NuGet.Protocol) to work with authenticated feeds.
+
+[!code-csharp[AuthenticatedFeed](~/../nuget-samples/NuGetProtocolSamples/Program.cs?name=AuthenticatedFeed)]
```

Comment:
The code snippet is missing a backtick. See: https://github.com/NuGet/Samples/commit/b2309028817bb6b0bbc7723c8f101241acd99790#r45328129

Reply:
Ahh, I'll fix that.